api/desecapi/tests/test_totp.py

```diff
@@ -105,12 +140,28 @@ def test_workflow(self):
         # Anonymous verification only allowed for activation
         response = self.client.post(url, {"code": authenticator.at(now)})
         self.assertResponse(response, status.HTTP_401_UNAUTHORIZED)
+
+        # Token has not yet passed MFA, but GET'ing TOTP list is allowed
+        self.assertFalse(self.token.mfa)
```

Comment:
passes for `None`. Did you mean `self.token.mfa is None`?

Reply:
Correction: Did you mean `self.token is False`?

Reply:
yes. Done